models/index.d.ts

```diff
@@ -45,6 +45,7 @@ export declare type Visit = {
     readonly clothingBoy?: boolean | null;
     readonly clothingMen?: boolean | null;
     readonly sleepingBag?: boolean | null;
+    readonly food?: boolean | null;
```

Comment:
I'm wondering do we need a food items quantity?

Reply:
That's a good question. Shall we ask the client to clarify before pushing the changes?

Reply:
The client confirmed that food quantity is unnecessary on the 11/1 meeting.